packages/cli/src/commands/setup/i18n/i18n.js

```diff
@@ -19,56 +19,146 @@ export const builder = (yargs) => {
+          throw new Error(
+            'i18n config already exists.\nUse --force to override existing config.'
+          )
+        } else {
+          return writeFile(
+            path.join(getPaths().web.src, 'i18n.js'),
+            fs
+              .readFileSync(
+                path.resolve(__dirname, 'templates', 'i18n.js.template')
+              )
+              .toString(),
+            { overwriteExisting: force }
+          )
+        }
       },
     },
     {
-      title: "Adding locale file for 'site' namespace",
-      task() {
-        return writeFile(getPaths().web.src + '/locales/en/site.json')
+      title: 'Adding locale file for French...',
+      task: () => {
+        /**
+         * Make web/src/locales if it doesn't exist
+         * and write fr.json there
+         *
+         * Check if fr.json already exists.
+         * If it exists, throw an error.
+         */
+
+        if (!force && localesExists('fr')) {
+          throw new Error(
+            'fr.json config already exists.\nUse --force to override existing config.'
+          )
+        } else {
+          return writeFile(
+            path.join(getPaths().web.src, '/locales/fr.json'),
+            fs
+              .readFileSync(
+                path.resolve(__dirname, 'templates', 'fr.json.template')
+              )
+              .toString(),
+            { overwriteExisting: force }
+          )
+        }
       },
     },
     {
-      title: 'Adding import to App.js...',
+      title: 'Adding locale file for English...',
       task: () => {
+        /**
+         * Make web/src/locales if it doesn't exist
+         * and write en.json there
+         *
+         * Check if en.json already exists.
+         * If it exists, throw an error.
+         */
+        if (!force && localesExists('en')) {
+          throw new Error(
+            'en.json already exists.\nUse --force to override existing config.'
+          )
+        } else {
+          return writeFile(
+            path.join(getPaths().web.src, '/locales/en.json'),
+            fs
+              .readFileSync(
+                path.resolve(__dirname, 'templates', 'en.json.template')
+              )
+              .toString(),
+            { overwriteExisting: force }
+          )
+        }
+      },
+    },
+    {
+      title: 'Adding import to App.js...',
+      task: (_ctx, task) => {
         /**
          * Add i18n import to the top of App.js
+         *
+         * Check if i18n import already exists.
+         * If it exists, throw an error.
          */
-        let indexJS = fs.readFileSync(INDEX_JS_PATH)
-        indexJS = [`import './i18n'`, indexJS].join(`\n`)
-        fs.writeFileSync(INDEX_JS_PATH, indexJS)
+        let appJS = fs.readFileSync(APP_JS_PATH)
+        if (i18nImportsExist(appJS)) {
+          task.skip('Import already exists in App.js')
+        } else {
+          appJS = [`import './i18n'`, appJS].join(`\n`)
```

Comment:
I understand what's happening here, but it feels strange to me to have an import with no reference or instantiation in the code. Maybe this is commonly done and best practice as is. But I might suggest including a comment explaining what's happening with the import.

Reply:
@dac09 I was curious about whether or not this works with pretender (server-side). And it does! I tested with the example i18n code for the Home page. Just an FYI.

Reply:
@thedavidprice I agree with you, but if we add a dependency to `import './index.css` and someone removes this file to have a like differents CSS files, he will not be able to make the i18n setup. What do you think?

Reply:
I find out that we can simply add it two lines up of ` const App = () => (`

Reply:
@simoncrypta sorry I missed this comment until now. 
> I find out that we can simply add it two lines up of const App = () => (

^^ This is a better approach I believe. It could also have a fallback to add at top of the file.

I'd like to see this merged for the version 0.27.0 release tomorrow. Would you be able to make this update today? 

Note: I just handled a merge conflict due to the Redwood new project template source now being TypeScript. This means `App.js` could also be `App.tsx`, so now there's a new helper to check `getPaths().web.app`

Reply:
@thedavidprice I will do my best to make this update today after my day job!

Reply:
Thanks, just keep me posted either way. I'm happy to help if/as needed.

If you do make the update, please do let me know this is ready!

Reply:
It was a cool challenge!